ts/node/migration/sessionMigrations.ts

```diff
@@ -1204,6 +1205,71 @@ function updateToSessionSchemaVersion29(currentVersion: number, db: BetterSqlite
   console.log(`updateToSessionSchemaVersion${targetVersion}: success!`);
 }
 
+function updateToSessionSchemaVersion30(currentVersion: number, db: BetterSqlite3.Database) {
+  const targetVersion = 30;
+  if (currentVersion >= targetVersion) {
+    return;
+  }
+
+  console.log(`updateToSessionSchemaVersion${targetVersion}: starting...`);
+
+  db.transaction(() => {
+    // Conversation changes
+    db.prepare(
+      `ALTER TABLE ${CONVERSATIONS_TABLE} ADD COLUMN expirationType TEXT DEFAULT "off";`
+    ).run();
+
+    db.prepare(
+      `ALTER TABLE ${CONVERSATIONS_TABLE} ADD COLUMN lastDisappearingMessageChangeTimestamp INTEGER DEFAULT 0;`
+    ).run();
+
+    db.prepare(`ALTER TABLE ${CONVERSATIONS_TABLE} ADD COLUMN hasOutdatedClient TEXT;`).run();
+
+    // TODO update to agreed value between platforms
+    const disappearingMessagesV2ReleaseTimestamp = 1706778000000; // unix 01/02/2024 09:00
+    // const disappearingMessagesV2ReleaseTimestamp = 1677488400000; // unix 27/02/2023 09:00
```

Comment:
We use window.log.info for logging inside of releaseFeature.ts which means we can't use that option. I can swap it for console logic or do a check for if window is defined and otherwise use console log?

Reply:
what I meant was to create a new file named releaseFeatureTimestamp.ts with just the exported timestamp in it.
You could then import that exported timestamp from the two different files just fine, even if one is using the window object and the other one not. I think?

Reply:
No worries. I have moved it into the constants file and future proofed it. Test and works

```
export const FEATURE_RELEASE_TIMESTAMPS: Record<string, number> = {
  // TODO update to agreed value between platforms
  DISAPPEARING_MESSAGES_V2: 1706778000000, // unix 01/02/2024 09:00
  // NOTE for testing purposes only
  // DISAPPEARING_MESSAGES_V2: 1677488400000, // unix 27/02/2023 09:00
};
```